Octokit.Tests/Clients/RepositoriesClientTests.cs

```diff
@@ -211,6 +211,158 @@ public class TheCreateMethodForOrganization
             }
         }
 
+        public class TheTransferMethod
+        {
+            [Fact]
+            public void EnsuresNonNullArguments()
+            {
+                var connection = Substitute.For<IApiConnection>();
+                var client = new RepositoriesClient(connection);
+                var transfer = new RepositoryTransfer("newOwner");
+
+                Assert.ThrowsAsync<ArgumentNullException>(
```

Comment:
All of these tests are actually not running properly because `Assert.ThrowsAsync` must be awaited